### DISCUSSION TITLE
How to configure custom domain as default for production deployment

### DISCUSSION THREAD

**Discussion by phuctm97** [vercel/vercel#4798] [Help]
Hi Vercel team,

I'm using Vercel to deploy my website with a custom domain, it works perfectly. However, the domain `xxx.vercel.app` is still considered default domain for my production deployment. Whenver I click on `View deployment` on Github it points to `xxx.vercel.app`, Lighthouse integration also points to that URL. Is there anyway I can configure a custom domain to be default for production deployment?

Thank you!

**Comment by paulogdm** [comment]
Hello,

You can remove the domain on your custom's domain tab. For further assistance, please reach out to https://vercel.com/support/request.
